web/app/forms.py

```diff
@@ -47,3 +54,17 @@ def clean(self):
             except PushbulletError:
                 self.add_error('pushbullet_access_token',
                                'Invalid pushbullet access token.')
+
+        if not data['telegram_chat_id']:
+            data['telegram_chat_id'] = None
+
+        if bot and data['telegram_chat_id']:
+            auth = json.loads(data['telegram_chat_id'])
+            data['telegram_chat_id'] = auth['id']
+            telegram_chat_id = data['telegram_chat_id']
```

Comment:
I think these 2 lines can be simplified as `telegram_chat_id =  auth['id']`.

Reply:
👍 Didn't know if you needed to explicitly set data attributes on a form for Django to save them out with those. Thanks.